frontend/controller/actions/group.js

```diff
@@ -714,19 +716,12 @@ export default (sbp('sbp/selectors/register', {
     })
   }),
   ...encryptedAction('gi.actions/group/removeMember',
-    (params, e) => L('Failed to remove {member}: {reportError}', { member: params.data.member, ...LError(e) }),
+    (params, e) => params.data.member ? L('Failed to remove {member}: {reportError}', { member: params.data.member, ...LError(e) }) : L('Failed to leave group. {codeError}', { codeError: e.message }),
     async function (sendMessage, params, signingKeyId) {
       await sendMessage({
         ...omit(params, ['options', 'action'])
       })
     }),
-  ...encryptedAction('gi.actions/group/removeOurselves',
-    (e) => L('Failed to leave group. {codeError}', { codeError: e.message }),
-    async function (sendMessage, params) {
-      await sendMessage({
-        ...omit(params, ['options', 'action'])
-      })
-    }),
```

Comment:
Why did you decide to remove this?

See also other comment below:

> Before, contracts could not remove someone else from another group. Now they can.

Reply:
I added this back but calling `removeMember`. As for removing it from the _contract_, it was to keep things DRY.